LICENSE

```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```

Comment:
@Natim is still in AUTHORS: https://github.com/novafloss/check_po/blob/master/AUTHORS